parcels/kernels/TEOSseawaterdensity.py

```diff
@@ -6,6 +6,7 @@
 
 
 def PolyTEOS10_bsq(particle, fieldset, time):
+    '''
     # calculates density based on the polyTEOS10-bsq algorithm from Appendix A.2 of
     # https://www.sciencedirect.com/science/article/pii/S1463500315000566
```

Comment:
Good to change to docstring, but the `#`s are then not necessary anymore?

Reply:
Yes, good point!